### PR TITLE
#167727759 users or managers should see most traveled destinations

### PR DESCRIPTION
**Pull request by dinorhythms**
#### What does this PR do?
This PR allows users and manager to see most traveled destinations
#### Description of Task to be completed?
- create migration file
- create model
- create route and add to index route
- create controller
- create services file
- add services to request controller
- test created endpoints
#### How should this be manually tested?
- clone the repo
- install dependencies
- install databases and configure
- create .env file for environment variables
- run npm run db:migrate && npm run seed
- POST /api/v1/request/accept/requestId 
- GET /api/v1/mostdestination
#### What are the relevant pivotal tracker stories?
[#167727759](https://www.pivotaltracker.com/story/show/167727759)

### CODE REVIEW

File: src/test/routes/mostdestination..spec.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  app, chai, expect, sinon, BACKEND_BASE_URL
+} from '../testHelpers/config';
+import mockData from '../mockData';
+import models from '../../models';
+import authHelper from '../../utils/authHelper';
+
+const { generateToken } = authHelper;
+const { User, Mostdestination } = models;
+
+// const {
```
Comment (tobslob) [inline src/test/routes/mostdestination..spec.js]: Nice work bro, remove commented code

Comment (dinorhythms) [reply src/test/routes/mostdestination..spec.js]: Thanks, i forgot to remove it

Comment (dinorhythms) [reply src/test/routes/mostdestination..spec.js]: It's been removed

**Review by tejiri4 — APPROVED** [review]
LGTM

**Pull request merged**